qt/scientific_interfaces/Indirect/ISISCalibration.cpp

```diff
@@ -50,7 +51,8 @@ ISISCalibration::ISISCalibration(IndirectDataReduction *idrUI, QWidget *parent)
   m_uiForm.ppResolution->setCanvasColour(QColor(240, 240, 240));
   m_uiForm.ppCalibration->watchADS(false);
   m_uiForm.ppResolution->watchADS(false);
-
+  m_uiForm.leScale->setValidator(new QRegExpValidator(QRegExp("\\d+(\\.\\d*)?")));
+  m_uiForm.leResolutionScale->setValidator(new QRegExpValidator(QRegExp("\\d+(\\.\\d*)?")));
```

Comment:
Not for this release, but this `.` (literal) matcher should be replaced with a `.` (for any char here) matcher

Since different locales use `,` as the decimal point separator on the UX side and I think this would force them to use `1.23` instead of `1,23`